### PR TITLE
fix model injection

### PR DESCRIPTION
**Pull request by IgorDePaula**

### CODE REVIEW

**Comment by Cayan** [comment]
Can you please remove the IDE files?

File: module/Application/src/Factory/Controller/ContactFactory.php [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    public function createServiceWithName(ServiceLocatorInterface $serviceLocator, $name, $requestedName)
+    {
+        $class = new \ReflectionClass($requestedName);
+        $parentLocator = $serviceLocator->getServiceLocator();
+        if( $constructor = $class->getConstructor() )
+        {
+            if( $params = $constructor->getParameters() )
+            {
+                $parameter_instances = [];
+                var_dump($params);exit;
```
Comment (Cayan) [inline module/Application/src/Factory/Controller/ContactFactory.php:38]: var_dump here

File: module/Application/view/application/contact/index.phtml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: igor
+ * Date: 10/11/16
+ * Time: 11:42
+ */
+echo __FILE__;
```
Comment (Cayan) [inline module/Application/view/application/contact/index.phtml:8]: Can you remove this file as well?

**Review by Cayan — CHANGES_REQUESTED** [review]

**Comment by IgorDePaula** [comment]
- Can’t automatically merge. \* Don’t worry, you can still create the pull
  request.

I tried remove it, but I got.

2016-11-10 15:13 GMT-02:00 Daniel Pinto notifications@github.com:

**Comment by Cayan** [comment]
Rebase to fix those conflicts.

**Pull request closed without merging**